.github/workflows/callable-e2e-runner.yml

```diff
@@ -35,19 +35,25 @@ jobs:
       framework: ${{ matrix.integ-config.framework }}
       category: ${{ matrix.integ-config.category }}
       sample_name: ${{ matrix.integ-config.sample_name }}
-      spec: ${{ matrix.integ-config.spec }}
+      spec: ${{ matrix.integ-config.spec || '' }}
       amplifyjs_dir: ${{ matrix.integ-config.amplifyjs_dir || false }}
       browser: |
         ${{
-          fromJson(
-            needs.e2e-prep.outputs.integ-utils
-          )[matrix.integ-config.browser]
+          matrix.integ-config.browser
           &&
-          toJSON(
+          (
             fromJson(
               needs.e2e-prep.outputs.integ-utils
             )[matrix.integ-config.browser]
+            &&
```

Comment:
Def worth revisiting all of this.  Wonder if there's a better way to factor this. Could see security complaining only because its getting overcomplicated.